tests/core/speechrecognition/locales.js

```diff
@@ -0,0 +1,26 @@
+import { transformSpeechRecognitionLocaleForEdge } from '../../../src/core/speechrecognition/locales';
+
+jest.mock('../../../src/core/constants', () => ({
+  SPEECH_RECOGNITION_LOCALES_SUPPORTED_BY_EDGE:
+    require('../../../conf/i18n/constants').SPEECH_RECOGNITION_LOCALES_SUPPORTED_BY_EDGE
```

Comment:
this is a cool way of doing it

Reply:
I just wanted all the locale stuff to be colocated - I'm a little paranoid about us forgetting to test new locales for SpeechRecognition